components/BookSection.js

```diff
@@ -0,0 +1,24 @@
+import Book from '../modules/Book.js';
+
+export default function displayBook(book) {
```

Comment:
- [ ] As asked in the project requirements we have to use ES6 syntax, so kindly use an arrow function instead of the function keyword

![image](https://user-images.githubusercontent.com/73140461/148850077-4b556424-f423-4465-a31b-c1bbe6783ad2.png)